server/data_cxg/cxg_adaptor.py

```diff
@@ -195,12 +195,7 @@ def _open_array(uri, tiledb_ctx):
     def open_array(self, name):
         try:
             p = self.get_path(name)
-            # TODO: remove this temporary workaround when TileDB fixes the issue with multi_index
-            if name == "X" or name == "X_col_shift":
-                # workaround: bypass the cache for arrays that use multi_index
-                return CxgAdaptor._open_array(p, self.tiledb_ctx)
-            else:
-                return self.arrays[p]
+            return self.arrays[p]
```

Comment:
Undoes this:
* https://github.com/chanzuckerberg/single-cell-explorer/pull/58/files#diff-489d8ec53771e2a834181354fec38d21a6e7cd0eb4dacb84265edd276323ed71R198